### PR TITLE
braces: replaced mysql by mariadb

### PR DESCRIPTION
**Pull request by D3vil0p3r**
MariaDB is the official MySQL-related package. mysqlpackage creates conflicts with other tools.

### CODE REVIEW

File: packages/braces/PKGBUILD [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-pkgrel=2
+pkgrel=3
 pkgdesc='A Bluetooth Tracking Utility.'
 url='http://braces.shmoo.com/'
 groups=('blackarch' 'blackarch-bluetooth')
 license=('GPL')
-depends=('php' 'mysql')
+depends=('php' 'mariadb')
 arch=('any')
 source=("http://braces.shmoo.com/blackhat-${pkgver//./}.tar.gz")
```
Comment (noraj) [inline packages/braces/PKGBUILD:13]: The source is dead.

**Review by noraj — CHANGES_REQUESTED** [review]

**Comment by noraj** [comment]
https://github.com/BlackArch/blackarch/pull/3876/files

File: packages/braces/PKGBUILD (hunk 1 above)
Comment (noraj) [reply packages/braces/PKGBUILD:13]: https://github.com/BlackArch/blackarch/pull/3876/files

Comment (D3vil0p3r) [reply packages/braces/PKGBUILD:13]: Ok. So its removal will solve that conflicting issue anyway.

**Pull request closed without merging**